Security/src/RedisDataProtection/README.md

```diff
@@ -37,21 +40,21 @@ Upon startup, the app displays session information on the home page, something l
    ```shell
    cf target -o your-org -s your-space
    ```
-   - When using Redis for Tanzu Application Service or Tanzu for Valkey on Cloud Foundry:
+   - When using Redis for Tanzu Application Service:
      ```shell
-     cf create-service p.redis on-demand-cache sampleRedisService
+     cf create-service p-redis shared-vm sampleRedisService
      ```
-     or:
+  - When using Tanzu for Valkey on Cloud Foundry:
      ```shell
-     cf create-service p-redis shared-vm sampleRedisService
+     cf create-service p.redis vk-plan sampleRedisService
```

Comment:
`p-redis shared-vm` is a plan that is [included by default](https://techdocs.broadcom.com/us/en/vmware-tanzu/data-solutions/tanzu-for-valkey-on-cloud-foundry/10-1/valkey-on-cf/architecture-pp.html) with the tile

`p.redis vk-plan` is an operator-configured [on-demand plan](https://techdocs.broadcom.com/us/en/vmware-tanzu/data-solutions/tanzu-for-valkey-on-cloud-foundry/10-1/valkey-on-cf/architecture.html) that is specific to our environment. Maybe we should leave this line as it was

Reply:
I just synced up from the Connector readme. I'm not sure how you'd like this changed. Can you provide a suggestion?

Reply:
Should it be "your-plan" like for AWS?

Reply:
Ah, I didn't look at that today and glossed over the details before...

Probably both places should be something like this:
```
   - When using Tanzu for Valkey on Cloud Foundry:
     ```shell
     cf create-service p.redis your-plan sampleRedisService
     ```
     or:
     ```shell
     cf create-service p-redis shared-vm sampleRedisService
     ```
```

Let's just leave out the older variation so we don't have to come back in 6 months or whenever it goes out of support... there's some expectation of "look at what's actually in your environment" with all of these service instances

Reply:
That's roughly what I meant... There might not be an ideal answer here since both are technically supported right now and there's significant overlap, but I'm not sure we know (or care about) any differences. I do want to avoid saying "use this for the Redis tile" and "use this for the ValKey tile" because (for example) usage of the shared plan looks identical and is acceptable in either case. 
What do you think about a generic phrase like "When using a Tanzu service offering for Valkey or Redis" and just avoid the specifics?